engine.py

```diff
@@ -95,15 +95,33 @@ def post_app_init(self):
 
         from sgtk.platform.qt import QtCore
 
-        if not self.has_ui:
-            # no UI. everything after this requires the UI!
-            return
-        
         tk_houdini = self.import_module("tk_houdini")
         bootstrap = tk_houdini.bootstrap
+        
+        if not self.has_ui:
+            bootstrap.bootstrap(self.sgtk, self.context)
```

Comment:
Why do we now need to call bootstrap here?
It wasn't doing this before, is it required for the otls to load when not UI is present?

Reply:
I've merged this branch into another one, and I've left this change out. I've written tests around the otl loading and as far as I can see, running this on Windows and Mac with hython, the otls are loaded fine without.